core/storage/hamt/hamt_walk.cpp

```diff
@@ -305,7 +305,7 @@ namespace fc::storage::hamt {
       for (auto &item2 : node.items) {
         OUTCOME_TRY(flush(item2.second));
       }
-      OUTCOME_TRYA(item, ipld->setCbor(node));
+      OUTCOME_TRYA(item, fc::setCbor(ipld, node));
```

Comment:
Why fc:: if you already in fc ?

Reply:
`Hamt::setCbor` shadows `fc::setCbor`, need to use namespaces

Reply:
Ok